decoder/pokemon.go

```diff
@@ -411,31 +411,27 @@ func (pokemon *Pokemon) isNewRecord() bool {
 	return pokemon.FirstSeenTimestamp == 0
 }
 
-func (pokemon *Pokemon) addWildPokemon(ctx context.Context, db db.DbDetails, wildPokemon *pogo.WildPokemonProto, timestampMs int64, timestampAccurate bool) {
+func (pokemon *Pokemon) addWildPokemon(ctx context.Context, db db.DbDetails, wildPokemon *pogo.WildPokemonProto, timestampMs int64) {
```

Comment:
this method is still called from `updateFromWild` and from `updatePokemonFromEncounterProto`. So also in the `updateFromWild` we do not update the spawnpoint anymore. we only update spawnpoint in `decoder.main` with `spawnpoint.spawnpointUpdateFromWild` ?

Reply:
The idea is spawnpoints are not updated during pokemon processing; they are updated separately. So if we choose to delay, or disable wild pokemon scanning we still get spawnpoints.  
Are you saying that you see updates from within pokemon still?

Reply:
you mean updates in DB? did not checked

I only followed the workflow and checked usages of that method in Goland (impossible to see usages in github review itself) :)

Reply:
should we add a proper comment to the method call? so we don't confuse us in the future on our own? :)

Reply:
I am unsure what needs to be commented - it was confusing before, and a little bit ugly (that a pokemon update could change a spawnpoint).  Now the two things are separate it is easier to see what causes spawnpoint changes - pokemon now just references this data